workflows/prognostic_c48_run/tests/_regtest_outputs/test_regression.test_fv3run_diagnostic_outputs_schema[nudging].out

```diff
@@ -1,10 +1,10 @@
 xarray.Dataset {
 dimensions:
-	time = 2 ;
 	tile = 6 ;
-	z = 63 ;
-	y = 12 ;
+	time = 2 ;
 	x = 12 ;
+	y = 12 ;
+	z = 63 ;
```

Comment:
Inconsequential changes like this are fine.

Reply:
Thank you!! I was going nuts trying to think why the ConstantOutputPredictor output would have changed. I think this was it.